### PR TITLE
Utterances

### PR DESCRIPTION
**Pull request by Tom-van-Woudenberg**
Add instructions on:
- basurl
- primary branch
- utterances optiosn

### CODE REVIEW

File: book/basic-features/utterances.md [hunk 1]
```diff
@@ -4,10 +4,14 @@
 
 [Utterances 🔮](https://utteranc.es/) is a lightweight open-source widget which allows you, your colleages and your students to discuss stuff in a blog post in your book. It is build on GitHub issues, so requires a GitHub repository, although the book can be hosted anywhere (so also on GitLab).
 
-The [utterances website](https://utteranc.es/) clearly explains the required steps. One thing to take care of is how you want to do the issue-mapping with respect to multiple deployed versions (draft version, student version, archived versions):
- 
- - `Issue title contains page pathname`/`Issue title contains page URL` is useful if you want to isolate the comments on each of the branches.
- - `Issue title contains page title` is useful if you want to show the all comments on all different branches.
+The [utterances website](https://utteranc.es/) clearly explains the required steps. Three things to take care of are:
+1. With the current setup of the [deploy book workflow on GitHub](../external/deploy-book-workflow/README.md), this widget only works on the primary branch.
+2. The baseurl needs to be defined in the template:
```
Comment (xjulie0x) [inline book/basic-features/utterances.md]: in the config.yml file of the template*

Comment (xjulie0x) [reply book/basic-features/utterances.md]: and then how can they find the baseurl, the url of their repo

Comment (Tom-van-Woudenberg) [reply book/basic-features/utterances.md]: Ah, that should indeed be just config, I'll change it

Comment (Tom-van-Woudenberg) [reply book/basic-features/utterances.md]: Is it better now? @xjulie0x ?

**Review by xjulie0x — APPROVED** [review]

**Comment by Tom-van-Woudenberg** [comment]
@pietervandekerckhove, FYI, we've updated the instructions based on your implementation, see https://teachbooks.io/manual/basic-features/utterances.html

**Comment by pietervandekerckhove** [comment]
wow great guys! Thank you for the support, something to add is relevant for my case, as I understand it now, is that uterances will store the issues as issues on the book file, but perhaps you would like multiple discussions archived differently, this would require multiple markdown book files as I understand this, I would need to seperate my book into seperate files probably at a later stage?

**Comment by Tom-van-Woudenberg** [comment]
uterances will store the issues as issues ~~on the book file~~ in the repository
Yes, uterances allows for on discussion per page in the book

**Comment by pietervandekerckhove** [comment]
so how do I separate the pages? I will make a new discussion in the file so perhaps you can show me?

**Comment by Tom-van-Woudenberg** [comment]
you can add a new markdown file to the repo and add it to the `_toc.yml`

**Comment by Tom-van-Woudenberg** [comment]
If you need help on that --> @xjulie0x

**Comment by pietervandekerckhove** [comment]
ok thank you, I thought it would be easier to work in one markdown file until my final draft is ready? Otherwise you have to switch constantly between markdown files? Instead of scrolling in one long file or is there an easier way to switch between markdownfiles?

**Comment by Tom-van-Woudenberg** [comment]
You could try this: 
image
https://teachbooks.io/manual/workflows/edit_book.html

**Comment by Tom-van-Woudenberg** [comment]
Well actually, you can have multiple utterances widgets on one page using the option for specific number or keyword.
But anyway it is nice if you can work on multiple files

**Comment by xjulie0x** [comment]
you can work in multiple markdown files in vs code just as you would work in multiple tabs in your browser, @pietervandekerckhove did you see my suggestion for the toc?

**Pull request merged**